ddl/job_table.go

```diff
@@ -225,18 +224,35 @@ func (d *ddl) delivery2worker(wk *worker, pool *workerPool, job *model.Job) {
 			asyncNotify(d.ddlJobCh)
 			metrics.DDLRunningJobCount.WithLabelValues(pool.tp().String()).Dec()
 		}()
-		// we should wait 2 * d.lease time to guarantee all TiDB server have finished the schema change.
-		// see waitSchemaSynced for more details.
+		// check if this ddl job is synced to all servers.
```

Comment:
If MDL feature set off, I thought we still fallback to wait 2 * d.lease time to do schema sync?

Reply:
wk.waitSchemaSynced(d.ddlCtx, job, 2*d.lease), the logical is same. This 2*lease is only used to check if it is 0.